docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,274 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Cause Model Diagram
+-------------------
+.. image:: joint_model_diagram.svg
 
-   all of this
+Data Description
+----------------
+.. list-table:: **Abbreviations**
+   :widths: 5 10
+   :header-rows: 1
+
+   * - Label
+     - Definition
+   * - S
+     - susceptible
+   * - C
+     - with condition
+   * - TB
+     - tuberculosis
+   * - LTBI
+     - latent tuberculosis infection
+   * - AcTB
+     - active tuberculosis
+   * - DS-TB
+     - Drug-susceptible TB
+   * - MDR-TB
+     - Multidrug-resistant TB
+   * - XDR-TB
+     - Extensively drug-resistant TB
+   * - HIV
+     - human immunodeficiency virus
+   * - HIV+ DS-TB
+     - HIV/AIDS - Drug-susceptible TB
+   * - HIV+ MDR-TB
+     - HIV/AIDS - Multidrug-resistant TB
+   * - HIV+ XDR-TB
+     - HIV/AIDS - Extensively drug-resistant TB
+   * - prev
+     - prevalence
+   * - i
+     - incidence
+   * - csmr
+     - cause-specific mortality rate
+   * - emr
+     - excess mortality rate
+   * - dw
+     - disability weight
+   * - {measure}_{gbd_id_type}{gbd_id}
+     - e.g. prev_c954 = prevalence of LTBI
+
+.. list-table:: **State Measures**
```

Comment:
oh boy 😂 ; it looks like your models were a test case we should've been using when selecting formatting. I think your equations are too wide, and now this table is compiling off the side of the page. I think you could either change your notation system, or we should reformat the CSS @kiranklc 

The other thing i would try is just defining less-wide columns than you currently have, and see to what degree it helps. It seems like (someone correct me if they know better) even when i've defined them quite small, they've expanded to accommodate the text I write.